### ISSUE TITLE
[Scene Composer] Save button is useless

### ISSUE THREAD

**Issue by grizeldi** [jMonkeyEngine/sdk#143]
Like really. Pretty much whatever I do in scene composer, especially if I'm editing terrain, save button will not be enabled. In most cases save all button will be enabled, however after closing the tab with scene composer even if you clicked save all, it still asks you if you want to save since it wasn't saved.
It got so bad I don't even bother with save all and just close and then select save. That seems to work properly at least, but it is **really** annoying. And this issue has been there for literally years.

**Comment by neph1** [comment]
I've noticed this, and have also recently tackled the same problem in the Macaq Editor. I might have a go at this.

**Comment by MeFisto94** [comment]
I just discovered something.
I changed (rotated) a light. Then I had the bug as described.

Selecting a Node in SceneExplorer made it possible to save as expected though.
Could it be that in your case (terrain, lights) it's the same?

**Comment by grizeldi** [comment]
Will have to try, but that's as annoying to do as just closing the tab and saving via popup.

**Comment by MeFisto94** [comment]
That's true and that's not how I meant this, but it might help narrowing down the issue (as in "we're setting the light as current object, not the j3o" or "the light is not seen as belonging to the j3o, where a mesh is")

**Comment by MeFisto94** [comment]
Note this is also related to #67 and #68
